run

```diff
@@ -281,11 +281,6 @@ build_local_libcucim_() {
 
     pushd "${source_folder}" > /dev/null
 
-    # Copy cufile SDK from host system to temp/cuda
-    copy_gds_files_ "$source_folder"
-    # Copy libjpeg SDK from host system to temp/cuda
-    copy_nvjpeg_files_ "$source_folder"
```

Comment:
This is how we installed the cuFile and nvJPEG when we needed them and they were either not in the CTK or the CTK versions were too old. However this is no longer an issue with CUDA 12.0+

Hence we are able to drop these and the two function definitions below